powerhome/playbook
# [PLAY-575] Added Position Fixed to Rails Dialog

#### Screens

![Screenshot 2023-01-26 at 11 09 08 AM](https://user-images.githubusercontent.com/73710701/214887340-97ac6c99-907c-4f62-bee3-b3e9f2477707.png)

#### Breaking Changes

No breaking change.

#### Runway Ticket URL

[Runway Story](https://nitro.powerhrg.com/runway/backlog_items/PLAY-575)

#### How to test this

Code only effects class wrapping the rails dialog kit, not react. Test all doc examples on rails dialog page in playbook to make sure change doesn't break anything. Fix however can only be tested in Nitro as the bug is coming from a CSS file in Nitro that is interfering with our styles. 

Will update here will alpha review env once it is ready.

#### Checklist:
